app/scripts/services/wsService.js

```diff
@@ -126,16 +126,11 @@ wsService.factory('searchService', ['$http', 'ENV', function($http, ENV){
           });
       },
       findGeneProducts: function(searchTerm, limit, page, facet, filters) {
-        return $http.get(ENV.apiEndpoint + '/geneproduct/search',
-          {
-            params: {
-              query : searchTerm,
-              limit : limit,
-              page : page ? page : 1,
-              facet : facet ? facet : '',
-              filterQuery : filters ? filters : ''
-            }
-          });
+        filters = filters.replace(/:/g, '=');
```

Comment:
This is causing the megasearch to break as there is no filters object. Why do we need to do this, can we just use the same syntax as expected by the backend?